build.gradle

```diff
@@ -43,11 +47,11 @@ subprojects {
 
   //noinspection GroovyAssignabilityCheck
   dependencies {
-    compile 'com.fasterxml.jackson.core:jackson-core:2.4.2'
-    compile 'com.fasterxml.jackson.core:jackson-annotations:2.4.2'
-    compile 'com.fasterxml.jackson.core:jackson-databind:2.4.2'
+    compile "com.fasterxml.jackson.core:jackson-core:$jacksonVersion"
```

Comment:
Minor issue here with the quotes using double quotes instead of single quotes that are used elsewhere. For consistency let's use single quotes 👍

Reply:
the double quotes are necessary to allow string interpolation to happen - single quotes mean the text is literal 

http://docs.groovy-lang.org/latest/html/documentation/#_string_interpolation